pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -135,6 +145,12 @@ func (r *ReconcileHyperConverged) Reconcile(request reconcile.Request) (reconcil
 		NetworkAddonsImagePullPolicy = instance.Spec.NetworkAddonsImagePullPolicy
 	}
 
+	if instance.Spec.KWebUIImagePullPolicy != "" {
```

Comment:
Can remove entire `if` block

Reply:
ok, make sense considering the "no option" operator case.
Is there a PR removing these options for other components as well?

Reply:
there will be

Reply:
removed